department-of-veterans-affairs/vets-website
# Pension 69970 mvp dependent disabled information

## Summary
On the Pension Benefits application(21P-527EZ), update the 'Health and employment information' chapter

## How to run in local environment
1. Check out this branch locally
2. In your local environment, set the `pension_form_enabled` flipper to 'enabled' at http://localhost:3000/flipper/features/pension_form_enabled
3. Run `vets-website` and `vets-api`

## How to verify
1. Go to `http://localhost:3001/pension/application/527EZ` in your browser
2. Start or continue the application
3. Go to the 'Household Information' chapter
4. Continue to the 'Dependent Children' page
5. Select 'Yes' and enter the required fields to add a dependent and continue
6. Verify that the 'Is your child seriously disabled?' question has a collapsed accordion
7. The accordion title is 'What do we mean by seriously disabled?'
8. Confirm that the accordion expands to reveal more information when you click the item
9. Confirm that the space button collapses the accordion
10. Verify the layout and content

## Related issue(s)
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/69970)
[Epic in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/63349)

## Testing done

## Screenshots

### Desktop
| Before | Dependent Information  | Dependent Information (Expanded)
| ------ | ------ | ------ |
|![localhost_3001_pension_application_527EZ_household_dependents_children_information_0](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/23a16635-028a-464d-baff-672ded070add)|![localhost_3001_pension_application_527EZ_ (1)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/5e6b67ce-c074-4017-bad9-ae5187d0f6d0)|![localhost_3001_pension_application_527EZ_ (2)](https://github.com/department-of-veterans-affairs/vets-website/assets/7394764/291d4362-aa65-4e06-a1a4-17ce28bea5b6)




## What areas of the site does it impact?
Pension Benefits Application

## Acceptance criteria
[Ticket in Github](https://github.com/department-of-veterans-affairs/va.gov-team/issues/69970)

### Quality Assurance & Testing

### Error Handling


### Authentication
